### PR TITLE
develop: enhance docs with workflow from customization exp.

### PR DESCRIPTION
**Pull request by fenekku**
- also add metadata reference to main page
- contribute section coming in another pr

### CODE REVIEW

File: docs/develop/edit_a_module.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If it is a backend change, simply installing the module from the local path
 should be enough. In some corner cases, you might have to re-start the
 instance (for that run `stop` and `run` again). For frontend (e.g. templates,
 css, etc.) you need to symlink and re-build the assets. This can be a complex
-process when using webpack, but Invenio-CLI got you covered.
+process when using webpack, but Invenio-CLI has got you covered.
```
Comment (ppanero) [inline docs/develop/edit_a_module.md:11]: minor: this doesnt sound goot to me, either `got` either `has`. But I'm not native 😅

File: docs/develop/edit_a_module.md [hunk 2]
```diff
@@ -45,15 +47,15 @@ invenio-cli assets watch
     changes. In addition, **be aware of the cache**, it might be a good idea
     to disable your cache or use an incognito window when developing web UI.
 
-This is a long running operation, so do not close the terminal. You
+This is a continually running operation, so do not close the terminal. You
```
Comment (ppanero) [inline docs/develop/edit_a_module.md:50]: minor:

```suggestion
This is a continuous running operation, so do not close the terminal. You
```

File: docs/develop/make_it_your_own.md [hunk 3]
```diff
@@ -65,37 +59,51 @@ If your logo isn't an svg, you still copy it to `static/images/`, but you need t
...
 !!! info "Re-run when invenio.cfg changes"
     All changes to `invenio.cfg` **MUST** be accompanied by a restart like the above to be picked up. This only restarts the server; it does not destroy any data.
 
-All other changes below follow the same pattern: add files and/or edit `invenio.cfg` and
-potentially execute `assets update` and/or rerun the server.
+This workflow stands for all `static/` files:
+
+- if you add a new file, run `invenio-cli assets update -d`
+- if you modify `invenio.cfg`, re-run `invenio-cli run` (because `invenio.cfg` has been symlinked above, you don't need to run `assets update`)
+- if you modify a previously symlinked file, you don't need to do anything
```
Comment (ppanero) [inline docs/develop/make_it_your_own.md:81]: 🚀 This a really good and necessary clarification!

File: docs/develop/make_it_your_own.md [hunk 4]
```diff
@@ -65,37 +59,51 @@ If your logo isn't an svg, you still copy it to `static/images/`, but you need t
...
+
+Open the `assets/less/variables.less` file and edit it as below:
 
 ``` less
 @navbar_background_image: unset;
 @navbar_background_color: #000000; // Note this hex value is an example. Choose yours.
 ```
 
-Then, run the `invenio-cli assets update` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available styling variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/variables.less).
+Then, run the `invenio-cli assets update -d` command as above and refresh the page! You should be able to see your top header's color changed! You can find all the available styling variables that you can change [here](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/variables.less).
```
Comment (ppanero) [inline docs/develop/make_it_your_own.md:97]: comment: this also imports the variables from other modules e.g. invenio-rdm-records, but I guess is not needed to say that some variables might come from other modules.

**Review by ppanero — APPROVED** [review]

File: docs/develop/edit_a_module.md (hunk 1 above)
Comment (fenekku) [reply docs/develop/edit_a_module.md:11]: Yeah, it's an informal idiom: https://www.merriam-webster.com/dictionary/have%20%28got%29%20someone%20or%20something%20covered ... I will change to just "has" to make it formal.

File: docs/develop/edit_a_module.md (hunk 2 above)
Comment (fenekku) [reply docs/develop/edit_a_module.md:50]: "continually" is the adverb applied to "running" (https://www.wordnik.com/words/continually).

File: docs/develop/make_it_your_own.md (hunk 4 above)
Comment (fenekku) [reply docs/develop/make_it_your_own.md]: That's a good point. I've enhanced it a little.

(I had originally just added `-d` but GitHub highlights the whole line :sweat_smile: )

**Review by fenekku — COMMENTED** [review]
Thanks! I've updated the doc.

**Comment by ppanero** [comment]
Thanks for the responses :) I've learnt something more today 🎊 Feel free to merge (same for the creatibutors PR)

**Pull request merged**